wordle-web/src/views/AboutView.vue

```diff
@@ -1,23 +1,56 @@
 <template>
-  <div class="about">
-    <v-card>
-      <v-card-title> This is a vue/vuetify class! </v-card-title>
-      <v-card-subtitle> (College Class) </v-card-subtitle>
-
-      <v-card-text>
-        Lorem ipsum dolor sit amet blah blah blah
-        <v-icon icon="mdi:mdi-abacus" />
-      </v-card-text>
-
-      <v-card-actions>
-        <v-spacer />
-        <v-btn text color="warning">Cancel</v-btn>
-        <v-btn text color="primary">OK</v-btn>
-      </v-card-actions>
-    </v-card>
-  </div>
+  <v-card class="mx-auto mt-12" max-width="844">
```

Comment:
You guys have lots of changes unrelated to the assignment. While your site looks great, it makes it difficult to read the git diff for the assignment. If you want to keep your custom looks and styles, consider using the second option for submitting PRs.